### ISSUE TITLE
Munki pkg descriptions dishonors newlines

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gonchik/munki#12]
```
What steps will reproduce the problem?
1. MunkiStatus.app launches and dishonors newlines in pkg description field.
2.
3.

What is the expected output? What do you see instead?
I would like to see newlines so the description field is more readable.

What version of the product are you using? On what operating system?
0.5.2.572, 10.6.x

Please provide any additional information below.
My pkginfo-files are UTF-8, editing them TextWrangler, using UNIX-newlines.
Should I use some special code to get munki showing newlines?
```

Original issue reported on code.google.com by `miq...@gmail.com` on 21 Jul 2010 at 9:10

**Comment by GoogleCodeExporter** [comment]
```
Assume you mean Managed Software Update.app.

Confirmed for Managed Software Update.app; MSU.app is using a WebKitView to 
display HTML descriptions coming from Apple SoftwareUpdate.  Probably need to 
detect when we have plain text and reformat accordingly, or see if I can tag 
the string as plain text instead of HTML.
```

Original comment by `gregnea...@mac.com` on 21 Jul 2010 at 9:25
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 3.0.1 of Managed Software Update.app, which is included with 
the 0.6.0.600.0 release of the munki tools.
```

Original comment by `gregnea...@mac.com` on 23 Jul 2010 at 12:03
- Changed state: **Fixed**
